### DISCUSSION TITLE
Calculate transaction time for TPROC-C workload

### DISCUSSION THREAD

**Discussion by Ora-DBA** [TPC-Council/HammerDB#608] [Q&A] [answered]
Hi, 
I'm performing test for TPROC-C workload. Test got executed successfully and got result of NOPM, TPM values. Is there any way we can track/calculate each transaction time (or time taken for single transaction) and what actual transaction or query or how much data is written to disk?  

HammerDB: 4.8
Workload: TPROC-C (OLTP), 100000 transaction per user, 1 VU, 5 min time
DB server: Oracle 19c 
DB client: 19.3

Thanks!

**Comment by sm-shaw** [comment, marked as answer]
Yes you can use time profiling as described here https://www.hammerdb.com/docs/ch04s07.html#d0e2189 by selecting the Time Profile check-box - also if you use the webservice to view your results it will automatically graph them for you: 

![Screenshot-at-2023-06-20-16-50-33-768x560](https://github.com/TPC-Council/HammerDB/assets/38044085/f80e61cd-b000-438b-920b-8d15d2210635)
